### ISSUE TITLE
Effective and trustworthy conformance evaluation for WCAG and beyond

### ISSUE THREAD

**Issue by hidde** [w3c/tpac2025-breakouts#28]
### Session description

[WCAG-EM](https://www.w3.org/TR/WCAG-EM/) is a process for deciding whether a website conform to [WCAG](https://www.w3.org/TR/WCAG22/), the Web Content Accessibility Guidelines, even if that website is very large. It is currently being updated (to [WCAG-EM 2](https://w3c.github.io/wai-wcag-em/). In this breakout, I'd like to:

- give an overview of what we're updating and solicit feedback 
- get input on making evaluation more effective
- get input on making evaluation more trustworthy

Slides: https://hidde.github.io/b6-logius/20251111-tpac-wcag-em/ 

[Slack: agwg-wcag-em](https://w3ccommunity.slack.com/archives/C08LDGQ16CD)

### Session goal

Share what's happening, solicit feedback

### Additional session chairs (Optional)

@JeroenHulscher

### Who can attend

Anyone may attend (Default)

### IRC channel (Optional)

[`#wcag-conformance`](https://webirc.w3.org/?channels=wcag-conformance)

### Agenda for the meeting.

- give an overview of what we're updating and solicit feedback 
- how can we make evaluation more effective
- how can we make evaluation more trustworthy

### Links to calendar

- [Wednesday, 14:45 - 15:45](https://www.w3.org/events/meetings/4313e40c-2ac1-4bfd-869d-16ac4e003e01/)

### Meeting materials

- [Minutes](https://www.w3.org/2025/11/12-wcag-conformance-minutes.html)
